Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM nginx:1.16.0
+ARG VERSION=1.16.0
+FROM nginx:${VERSION}
```

Comment:
Will this work for docker hub automated builds?

Reply:
AFAIK yes

Reply:
My bad, I've misread the diff